src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -75,6 +76,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case ListCommand.COMMAND_WORD:
             return new ListCommand();
 
+        case SortCommand.COMMAND_WORD:
+            return new SortCommandParser().parse(arguments);
```

Comment:
Is there a reason why you return this instead of  new SortCommand()?

Reply:
It is because unlike commands like `ListCommand` which takes in no arguments, the `SortCommand` takes in a prefix which refers to a particular attribute in `Person` objects to be sorted by.